### ISSUE TITLE
zmwfilter exponentially slower with more zmw holes

### ISSUE THREAD

**Issue by gevro** [PacificBiosciences/pbbioconda#464]
Hi,
I've confirmed on four different machines, including with a high-speed SSD drive, that there is an issue with zmwfilter, in that it becomes exponentially slower once you start requesting tens of thousands of holes to retrieve. The issue is not noticeable with 1 or a few hundred holes. Only once you get to > 10,000 or so.

Also reported by another user: PacificBiosciences/zmwfilter#1

I don't think it is an issue with disk speed.

If the zmw holes are indexed, it should be much faster than a single pass through the subreads.BAM to retrieve them, but that does not seem to be the case.

Thanks.

**Comment by armintoepfer** [comment]
Duplicate of #461
